source/includes/steps-import.yaml

```diff
@@ -19,25 +21,57 @@ content: |
 
   .. figure:: /images/compass/import-data-dialog.png
      :alt: Import data dialog
+     :width: 60%
 
 ---
-title: Choose the appropriate file type of the source data file. 
+title: Select the location of the source data file under 
+       :guilabel:`Select File`.
+level: 4
+ref: click-file-location
+content: ""
+
+---
+title: Choose the appropriate file type.
 level: 4
 ref: choose-file-type
 content: |
-  Under :guilabel:`Select Input File Type`, select either JSON or CSV. 
+
+  Under :guilabel:`Select Input File Type`, select either 
+  :guilabel:`JSON` or :guilabel:`CSV`. 
+
+  If you are importing a CSV file, you may specify fields to import and 
```

Comment:
It was a little bit weird to see this content when I had selected the JSON tab in the previous section, but I see now that this content is not tabbed. Maybe it should be? I don't feel strongly either way, but something to think about since it didn't confuse me for a second.

Reply:
I agree that it's weird to see this at all, but will try to improve it along the lines of your next comment: differentiating the CSV-only and JSON + CSV options. The distinction is made in one of the YAML procedure files, so to tab it out I think I would need to move that content to the import-export.txt file. That's definitely doable if we decide it's worth it, but for the sake of the existing structure I'll leave as-is for now.

Reply:
Ok we can leave as-is.